themes-default/slim/src/store/modules/config.js

```diff
@@ -263,7 +263,7 @@ const mutations = {
 };
 
 const getters = {
-    layout: layout => state => {
+    layout: state => layout => {
```

Comment:
This isn't right. This a higher level function, the state is the most inner function.

Reply:
https://vuex.vuejs.org/guide/getters.html#method-style-access
```js
getters: {
  // ...
  getTodoById: (state) => (id) => {
    return state.todos.find(todo => todo.id === id)
  }
}
```

Reply:
I'm an idiot.